backend/cms/templates/_raw.html

```diff
@@ -13,11 +13,13 @@
     <link href="{% static 'css/style.min.css' %}" rel="stylesheet" type="text/css">
     <link href="{% static 'css/upload_form.css' %}" rel="stylesheet" type="text/css">
     <link href="{% static 'css/tailwind.min.css' %}" rel="stylesheet"type="text/css">
+    {% block css %}{% endblock %}
     {% endcompress %}
     {% compress js %}
     <script src="{% static 'umbrellajs/umbrella.js' %}"></script>
     <script src="{% static 'feather-icons/dist/feather.js' %}"></script>
     <script src="{% static 'base64-js/base64js.min.js' %}"></script>
+    {% block javascript_head %}{% endblock %}
     {% endcompress %}
 </head>
 <body>
```

Comment:
We should put the feather.replace() call to a nocompress block, as the compression leads to not showing any icons in the CMS

Reply:
`feather.replace()` is also compressed in the [current develop branch](https://github.com/Integreat/cms-django/blob/351398d3e617f4f5aa0d42d4a7216295652e10ed/backend/cms/templates/_raw.html#L26)... so I don't think that's the issue here...

Reply:
Yes, but now the compression differs in such way, that it ends up with ;feather.replace() instead of the semicolon on the other side

Reply:
Ok, this sounds like dark magic to me :fearful: